lib/action_controller/serialization.rb

```diff
@@ -27,6 +27,7 @@ def get_serializer(resource, options = {})
       end
       serializable_resource = ActiveModel::SerializableResource.new(resource, options)
       if serializable_resource.serializer?
+        serializable_resource.url_helper = self
```

Comment:
yeah, plus there's already a `serialization_scope` options, so maybe `serialization_context` would be good